### PR TITLE
added courses tab for admin

### PR DESCRIPTION
**Pull request by cbarb**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@ant-design/icons": "^4.0.2",
+    "@apollo/client": "^3.2.0",
```
Comment (andres-valdes) [inline frontend/package.json:7]: Remove

File: frontend/package.json [hunk 2]
```diff
@@ -50,7 +51,7 @@
     "@types/relay-runtime": "^8.0.6",
     "@types/styled-components": "^5.0.1",
     "babel-plugin-relay": "^8.0.0",
-    "graphql": "^14.6.0",
+    "graphql": "^14.7.0",
```
Comment (andres-valdes) [inline frontend/package.json:54]: Why?

Comment (cbarb) [reply frontend/package.json:54]: No idea how that even happened tbh, all i did was add the courses tab to the sidebar. Must have been while setting up the project, didn't seem to break it though.

**Review by HeIp — APPROVED** [review]

**Review by Eberm024 — APPROVED** [review]

**Review by deellah — APPROVED** [review]

**Review by alemasserrano — APPROVED** [review]

**Comment by HeIp** [comment]
Team decided we are no longer adding this

**Pull request closed without merging**